lib/web_console/templates/console.js.erb

```diff
@@ -370,4 +373,113 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.inner.scrollTop = this.inner.scrollHeight;
 };
 
+/**
+ * Install the console into the element with a specific ID.
+ * Example: REPLConsole.installInto("target-id")
+ */
+REPLConsole.installInto = function(id) {
+  var consoleElement = document.getElementById(id);
+  var replConsole = new REPLConsole({
+    promptLabel: consoleElement.dataset.initialPrompt,
+    commandHandle: function(line) {
+      var _this = this;
+      var url = consoleElement.dataset.remotePath;
+      var params = "input=" + encodeURIComponent(line);
+      putRequest(url, params, function(xhr) {
+        var response = JSON.parse(xhr.responseText);
+        _this.writeOutput(response.output);
+      });
+    }
+  });
+
+  replConsole.install(consoleElement);
+};
+
+// Hash object for console events.
+REPLConsole.events = {}
+
+/**
+ * Bind and trigger console events ("changeFrame" and etc...?).
+ */
+function ConsoleEvent() {
+  this.listeners = {}; // this.listeners[eventName]
+}
+
+ConsoleEvent.prototype.trigger = function(eventName, _args_) {
+  // Extract listener's args
+  var listenerArgs = Array.prototype.slice.call(arguments);
+  listenerArgs.shift();
+
+  var listeners = this.listeners[eventName];
+  if ( listeners instanceof Array ) {
+    // Call each listener with args
+    listeners.forEach(function(listener) {
```

Comment:
We may discuss dropping the support of old browsers based on, say, Turbolinks supported browsers. But until then, let's try to avoid native `forEach`.